tests/test_wrappers.py

```diff
@@ -3,20 +3,23 @@
 import unittest
 import os
 
+
 from plumbum import local
 from plumbum.cmd import rm
 
+from benchbuild.downloads.base import nosource
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No name 'downloads' in module 'benchbuild'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)